desktop/src/args.rs

```diff
@@ -39,3 +59,9 @@ fn parse_scale(src: &str) -> Result<Scale, &'static str> {
         _ => Err("only '1', '2', '4', '8', '16', '32' or 'fit' are allowed"),
     }
 }
+
+fn parse_breakpoint(src: &str) -> Result<Word, String> {
+    u16::from_str_radix(src, 16)
+        .map(Word::new)
+        .map_err(|e| format!("failed to parse breakpoint: {}", e))
```

Comment:
Is the `--help` message triggered here? If not, information about the proper format would be helpful.